tools/SetupFlow/DevHome.SetupFlow/Controls/SetupShell.xaml

```diff
@@ -29,12 +32,20 @@
                 Style="{ThemeResource SubtitleTextBlockStyle}"
                 TextWrapping="WrapWholeWords"
                 Visibility="{x:Bind UseOrchestratorTitle, Mode=OneWay}"
-                Text="{x:Bind Orchestrator.FlowTitle}" />
+                Text="{x:Bind Orchestrator.FlowTitle}">
+                <i:Interaction.Behaviors>
+                    <behaviors:TextBlockAutomationBehavior RaiseLiveRegionChangedEvent="True" LiveSetting="Assertive" />
+                </i:Interaction.Behaviors>
+            </TextBlock>
             <TextBlock
                 Style="{ThemeResource SubtitleTextBlockStyle}"
                 TextWrapping="WrapWholeWords"
                 Visibility="{x:Bind Title, Mode=OneWay, Converter={StaticResource StringVisibilityConverter}}"
-                Text="{x:Bind Title, Mode=OneWay}" />
+                Text="{x:Bind Title, Mode=OneWay}">
+                <i:Interaction.Behaviors>
+                    <behaviors:TextBlockAutomationBehavior RaiseLiveRegionChangedEvent="True" LiveSetting="Assertive" />

```

Comment:
Maybe I'm looking at the wrong place but when testing this, in the end to end flow it we don't read out the subtitle of the page, e.g End-to-End gets read out but not "Clone repositories" etc. Is that supposed to happen?

Reply:
The subtitle is not announced because the displayed text only contains the title. But we **should** announce the subtitle to better help guide the user. I added this to the follow up issue: #1450